src/main/java/gregtech/GregTechMod.java

```diff
@@ -57,7 +57,7 @@
 @Mod(modid = GTValues.MODID,
         name = "GregTech",
         acceptedMinecraftVersions = "[1.12,1.13)",
-        dependencies = "required:forge@[14.23.5.2847,);" + CodeChickenLib.MOD_VERSION_DEP + "after:forestry;after:jei@[4.15.0,);after:crafttweaker")
+        dependencies = "required:forge@[14.23.5.2847,);" + CodeChickenLib.MOD_VERSION_DEP + "after:forestry;after:jei@[4.15.0,);after:crafttweaker@[4.1.20,);")
```

Comment:
Which version of CT did we need to depend on, because 4.1.20+ is like 100 different releases

Reply:
This is about as granular version as CT reports